.github/actions/reuse-old-whl/reuse_old_whl.py

```diff
@@ -307,15 +309,19 @@ def parse_args() -> argparse.Namespace:
 
 
 def can_reuse_whl(args: argparse.Namespace) -> bool:
-    # if is_main_branch() or (
-    #     args.github_ref
-    #     and any(
-    #         args.github_ref.startswith(x)
-    #         for x in ["refs/heads/release", "refs/tags/v", "refs/heads/main"]
-    #     )
-    # ):
-    #     print("On main branch or release branch, rebuild whl")
-    #     return False
+    if args.github_ref and any(
+        args.github_ref.startswith(x)
+        for x in [
```

Comment:
Should this also include nightly?

Reply:
I'm under the impression that this optimization is only for CI build and nightly binaries are always built fresh